build.sbt

```diff
@@ -73,11 +73,30 @@ lazy val core = crossProject(JSPlatform, JVMPlatform)
   .settings(testFrameworks := Seq(new TestFramework("zio.test.sbt.ZTestFramework")))
   .enablePlugins(BuildInfoPlugin)
 
-lazy val coreJS = core.js
+lazy val coreJS     = core.js
   .settings(jsSettings)
+  .settings(libraryDependencies += "dev.zio" %%% "zio-test-sbt" % zioVersion)
 
-lazy val coreJVM = core.jvm
+lazy val coreJVM    = core.jvm
   .settings(dottySettings)
+  .settings(libraryDependencies += "dev.zio" %%% "zio-test-sbt" % zioVersion)
```

Comment:
Is it supposed to have 3 `%%%` even for JVM?

Reply:
No I think it can be `%%`, just copied it from the shared place. Although I'm not completely aware of how `CrossProject` modifies the behavior of `libraryDependencies`.